LykkeApi2/Controllers/FeeSettingsController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Lykke.Service.FeeCalculator.AutorestClient.Models;
+using Lykke.Service.FeeCalculator.Client;
+using LykkeApi2.Infrastructure;
+using LykkeApi2.Models.Settings;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LykkeApi2.Controllers
+{
+    [Authorize]
+    [Route("api/[controller]")]
+    [Produces("application/json")]
+    [LowerVersion(Devices = "android", LowerVersion = 961)]
+    public class FeeSettingController : Controller
+    {
+        private readonly IFeeCalculatorClient _feeCalculatorClient;
+
+        public FeeSettingController(IFeeCalculatorClient feeCalculatorClient)
+        {
+            _feeCalculatorClient = feeCalculatorClient ?? throw new ArgumentNullException(nameof(feeCalculatorClient));
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> Get(string assetId = null)
```

Comment:
Do we really need a method for all fees at once? If no, please consider modifying this method, if yes, please consider splitting it in two and returning a model in one case and an array in the other.